README.md

```diff
@@ -56,7 +56,7 @@ Monitoring daemon (via [/r/TranscribersOfReddit/new](https://www.reddit.com/r/Tr
 To build the package from source, start in the base of the repository and run:
 
 ```sh
-$ python setup.py bdist --format=gztar
+$ poetry build
```

Comment:
Since `poetry build` produces both the sdist and the wheel, we should specify that the `.whl` file should be the one that's used. It's far more strict, such as having to manually declare any data files that would be shipped with the package.

I found that one out pretty recently...